config/base/params.env

```diff
@@ -1,11 +1,11 @@
 IMAGES_DSPO=quay.io/opendatahub/data-science-pipelines-operator:latest
-IMAGES_APISERVER=quay.io/opendatahub/ds-pipelines-api-server:latest
-IMAGES_PERSISTENCEAGENT=quay.io/opendatahub/ds-pipelines-persistenceagent:latest
-IMAGES_SCHEDULEDWORKFLOW=quay.io/opendatahub/ds-pipelines-scheduledworkflow:latest
+IMAGES_APISERVER=quay.io/opendatahub/ds-pipelines-api-server:pr-134
```

Comment:
Is this tag change intentional?

Reply:
It's intentional, but it's using the wrong image. I'm glad you asked.
Let-me update it.

Reply:
@rimolive I updated the commit to point to the images from https://github.com/opendatahub-io/data-science-pipelines/pull/142.
Once we feel ready to merge I'll remove the commit.